### ISSUE TITLE
white screen

### ISSUE THREAD

**Issue by thewh1teagle** [r0x0r/pywebview#1198]
Hi
I use `pywebview` and it worked for the past few days normally.
but today I got this error:
```log
[pywebview] Using WinForms / Chromium
[pywebview] WebView2 initialization failed with exception:
  (0x8007139F): The group or resource is not in the correct state to perform the requested operation. (HRESULT: 0x8007139F)
    System.Runtime.InteropServices.Marshal.ThrowExceptionForHRInternal(Int32 errorCode, IntPtr errorInfo)
  Microsoft.Web.WebView2.Core.CoreWebView2Environment.<CreateCoreWebView2ControllerAsync>d__8.MoveNext()
 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
Microsoft.Web.WebView2.WinForms.WebView2.<InitCoreWebView2Async>d__25.MoveNext()
```
the window is opened with only white screen

OS: `Windows 11` 
Python version: `python3.8` 
Pywebview version: `4.2.2`

**Comment by r0x0r** [comment]
This issue provides some details on this error. Looks like it is related to DPI settings

https://github.com/MicrosoftEdge/WebView2Feedback/issues/2495

**Comment by thewh1teagle** [comment]
I also developed with tauri in rust which uses webview2 too and it works normally

**Comment by gsal** [comment]
I am back.

I guess the hints above helped me get up and running; but, I ended up discovering that it is not working correctly, after all.

I have access to two platforms Centos7 and Rocky9.

Same installation process in each platform (python 3.9):
```bash
python -m venv venv
source ./venv/bin/activate
python -m pip install -U pip
pip install pywebview[qt]               # today, so, 4.3.3
``` 

Then:
 - I placed centos7 and rocky9 side-by-side
 - Tried 3 pywebview examples
 - On both systems, the sample apps come up and seem to do what it was intended: pop a dialog, hide the window, etc.
 - But
   - in centos7, the main window of the app shows content (advertising pywebview, user-agent, support)
   - in rocky9, the main window of the app remains blank; no error messages even when using "debug=True"
